Source/Eto/Forms/Controls/Splitter.cs

```diff
@@ -34,6 +34,9 @@ public enum SplitterFixedPanel
 	[Handler(typeof(Splitter.IHandler))]
 	public class Splitter : Container
 	{
+		bool shown;
+		int panel1MinWidth, panel2MinWidth = -1;
```

Comment:
The default values for the minimum width should probably be 0 instead of -1, as -1 doesn't have any meaning for a minimum size.  Also, these may be better named as `Panel1MinimumSize` and `Panel2MinimumSize`, as I'd like to refrain from using abbreviated names and also 'Width' does not apply if it is vertical.